playwright-tests/tests/editor.spec.js

```diff
@@ -0,0 +1,232 @@
+import { expect, test } from "@playwright/test";
```

Comment:
I'm kinda confused why this pull request is introducing these changes? These are already on develop...?

Reply:
yeah, I m too, I already rebased it twice after your PR was merged.